docs/guides/server/reverseproxy.adoc

```diff
@@ -29,8 +29,10 @@ If this header is incorrectly configured, rogue clients can set this header and
 
 NOTE: When using the `xforwarded` setting, the `X-Forwarded-Port` takes precedence over any port included in the `X-Forwarded-Host`.
 
+NOTE: If the TLS connection is terminated at the reverse proxy (edge termination), enabling HTTP through the ‘http-enabled’ setting is required.
+
 == Proxy modes
-NOTE: The support for setting proxy modes is deprecated and will be removed in a future {project_name} release. Consider configuring accepted reverse proxy headers instead as described in the chapter above. For migration instructions consult the https://www.keycloak.org/docs/latest/upgrading/index.html#deprecated-proxy-option[Upgrading Guide].
+NOTE: The support for setting proxy modes is deprecated and will be removed in a future {project_name} release. Consider configuring accepted reverse proxy headers instead as described in the chapter above. Enabling HTTP via `http-enabled` is necessary if edge termination is used. For migration instructions consult the https://www.keycloak.org/docs/latest/upgrading/index.html#deprecated-proxy-option[Upgrading Guide].
```

Comment:
@MichaelWarnecke Thank you for the PR!

We usually document any migration steps in the [Upgrading Guide](https://www.keycloak.org/docs/latest/upgrading/index.html#deprecated-proxy-option). Could you please review that if that is perhaps sufficient?

Reply:
@vmuzikar I added this line mainly because #29665 asked for the clarification, the upgrade guide states what parameters should be used quite clearly.
Nontheless, I think the other added line (https://github.com/keycloak/keycloak/pull/30170/files#diff-4a1b33f7b0a6b8526caf3186df5ccd193f7efcb683dcff9e515de2765ec9fd19R32) helps people who first install keycloak without updating, as I had exactly the same issue and spend quite some time figuring out, setting `http-enabled=true` is a valid option. Since I did a fresh install, I obviously didn't read the update docs.

Reply:
@vmuzikar Agreed, I removed the line